prow/github/client.go

```diff
@@ -795,6 +824,16 @@ func (c *Client) ReopenIssue(org, repo string, number int) error {
 		requestBody: map[string]string{"state": "open"},
 		exitCodes:   []int{200},
 	}, nil)
+	requestErr, ok := err.(requestError)
```

Comment:
Instead of copying this code between `ReopenPR` and `ReopenIssue` you could move it to a function that accepts an error and returns an error of the appropriate type.

Reply:
done